### DISCUSSION TITLE
I want the mobile app to have the ability to switch accounts.

### DISCUSSION THREAD

**Discussion by okaryou7** [immich-app/immich#13743] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I want the mobile app to have the ability to switch accounts. To switch accounts, you need to sign out once. I would like a feature like Google Photos that allows you to switch to another account while signed in.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2919

As the template requests, please search for duplicates when you open an FR.
